src/app.ts

```diff
@@ -1,12 +1,19 @@
 import express from 'express';
 import swaggerJSDoc from 'swagger-jsdoc';
 import swaggerUi, { SwaggerUiOptions } from 'swagger-ui-express';
+import 'dotenv/config';
 
 import { healthCheckRoute } from './routes/api/health-check.js';
-import { coursesRoute } from './routes/api/courses-routes.js';
+import { coursesRouter } from './routes/api/courses-routes.js';
+import { moviesRouter } from './routes/api/movies-routes.js';
 import { generalErrorHandler } from './error-handlers/general-error-handler.js';
 import { notFoundHandler } from './error-handlers/not-found-handler.js';
 import swaggerOptions from './swagger/swagger-options.js';
+import connectDB from './config/db.js';
+import { genresRouter } from './routes/api/genre-routes.js';
+import { authorRouter } from './routes/api/author-routes.js';
```

Comment:
I need to use .js extension in my import because I use "module":"nodenext" in my tsconfig.json and "type":"module" in package.json. Should I swithch back to commonJS?

Reply:
Ok, let it be, but it's a not recommended to use edge technologies before they become stable. Such usage can lead to not correct work on different platforms or even some functionality may be removed in future for different reason. This may brake app without changes from developer side.